qa/crontab/teuthology-cronjobs

```diff
@@ -55,27 +54,6 @@ CEPH_QA_EMAIL="ceph-qa@ceph.io"
 
 ##########################
 
-#********** nautilus branch START - weekly
-
-25 13 * * 5  CEPH_BRANCH=nautilus; MACHINE_NAME=gibba; SUITE_NAME=kcephfs;  KERNEL=distro; /home/teuthology/bin/cron_wrapper /home/teuthology/bin/schedule_subset.sh 2999 $CEPH_BRANCH $MACHINE_NAME $SUITE_NAME $CEPH_QA_EMAIL $KERNEL
-15 05 * * 0  CEPH_BRANCH=nautilus; MACHINE_NAME=gibba; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 100 -m $MACHINE_NAME -s krbd -k testing -e $CEPH_QA_EMAIL
```

Comment:
As I said in https://github.com/ceph/ceph/pull/41226#pullrequestreview-654736621, I would like to keep the weekly `krbd` run for now.

Reply:
I thought we discussed it before `nautilus` EOL